coriolis/conductor/rpc/client.py

```diff
@@ -259,11 +259,17 @@ def task_event(self, ctxt, task_id, level, message):
         self._client.cast(ctxt, 'task_event', task_id=task_id, level=level,
                           message=message)
 
-    def task_progress_update(self, ctxt, task_id, current_step, total_steps,
-                             message):
-        self._client.cast(ctxt, 'task_progress_update', task_id=task_id,
-                          current_step=current_step, total_steps=total_steps,
-                          message=message)
+    def add_task_progress(self, ctxt, task_id, total_steps, message):
```

Comment:
Please update all of the names of these methods so that they're `*_task_progress_update` (note the `_update` suffix)